app/controllers/evaluations_controller.rb

```diff
@@ -25,6 +25,25 @@ def create
     @evaluation = Evaluation.new(evaluation_params)
 
     if @evaluation.save
+      if params["evaluation"]["evaluation_infos"].present? && ENV['ENABLE_AUTHENTICATION'].present?
+        infos = JSON.parse(params["evaluation"]["evaluation_infos"])
+        evaluation_mode = infos["evaluation_mode"]
+        questions = infos["questions"]
+        author = @evaluation.pia.user_pias.find_by({role: "author"}).user
+        evaluator = @evaluation.pia.user_pias.find_by({role: "evaluator"}).user
+        if evaluation_mode === 'item'
+          send_email_for_evaluator(evaluator, @evaluation.pia) if evaluator.present?
+          # Mail Sending
+        elsif evaluation_mode === 'question'
+          reference_to = @evaluation.reference_to.split(".")
+          if questions[0]["id"] == reference_to.last.to_i
+            send_email_for_evaluator(evaluator, @evaluation.pia) if evaluator.present?
+            # Mail Sending
+          end
+        end
+          
+      end
```

Comment:
@syl-p You should refactor this:
1. Add a method inside the Evaluation model
2. Use the after_save action
3. Do not repeat `send_email_for_evaluator`